bodhi/server/consumers/automatic_updates.py

```diff
@@ -156,6 +156,8 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
                 type=UpdateType.unspecified,
                 stable_karma=3,
                 unstable_karma=-3,
+                autokarma=False,
+                autotime=False,
```

Comment:
We need autotime otherwise the update will not get pushed to stable automatically. For rawhide the time threshold is set to 0 days :-).

I agree that autokarma is probably not needed

Reply:
okies, updated the commit, removing the autotime. thanks!